Features/Transfer/Sources/Scenes/AmountScene.swift

```diff
@@ -49,6 +49,13 @@ struct AmountScene: View {
                         )
                     }
                 }
+                
+                if let infoText = model.reservedFeesText {
```

Comment:
```suggestion
                if let infoText = model.infoText {
```

as we might add other warning messages